pkg/scheduler/plugins/deviceshare/device_plugin_adapter.go

```diff
@@ -118,10 +119,14 @@ func (a *generalDevicePluginAdapter) Adapt(object metav1.Object, _ []*apiext.Dev
 // generalGPUDevicePluginAdapter annotates necessary information to pod which resolves problems when using device plugin
 // with koord-scheduler.
 type generalGPUDevicePluginAdapter struct {
+	nodeName string
```

Comment:
It's better to make node information a context param to the adapt method rather then put it in adpter struct. In fact, I'm introducing such context in a recent work, maybe I can commit the context work in ahead so that this PR can reuse that? cc @ZiMengSheng

Reply:
agree